features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/responsive/templates/sso/filter/template.jag

```diff
@@ -54,15 +50,24 @@
             if(site.ssoConfiguration.keyStoreName == null || site.ssoConfiguration.keyStoreName == ""){
                 site.ssoConfiguration.keyStoreName = CarbonUtils.getServerConfiguration().getFirstProperty("Security.KeyStore.Location");
                 }
+            if(site.ssoConfiguration.signRequests == null || site.ssoConfiguration.signRequests == ""){
+                site.ssoConfiguration.signRequests = "false";
+            }
             ssoRelyingParty.setProperty("identityProviderURL", site.ssoConfiguration.identityProviderURL);
             ssoRelyingParty.setProperty("keyStorePassword", String(site.ssoConfiguration.keyStorePassword));
             ssoRelyingParty.setProperty("identityAlias", String(site.ssoConfiguration.identityAlias));
             ssoRelyingParty.setProperty("keyStoreName", String(site.ssoConfiguration.keyStoreName));
+            ssoRelyingParty.setProperty("signRequests", String(site.ssoConfiguration.signRequests));
 
+        var consumerUrl = jagg.getHttpsUrl("/jagg/jaggery_acs.jag");
         if(site.ssoConfiguration.passive == "true")    {
-            samlAuthRequest = ssoRelyingParty.getSAMLPassiveAuthRequest(isPassiveAuthRequired, acsUrl);
-        } else  {
-            samlAuthRequest = ssoRelyingParty.getSAMLAuthRequest();
+            samlAuthRequest = ssoRelyingParty.getSAMLAuthRequest(isPassiveAuthRequired, consumerUrl);
+        } else {
+            if(site.ssoConfiguration.signRequests && (site.reverseProxy.enabled == "auto")){
+                samlAuthRequest = ssoRelyingParty.getSAMLAuthRequest(consumerUrl);
+            }else{
+                samlAuthRequest = ssoRelyingParty.getSAMLAuthRequest();
+            }

```

Comment:
Fixed in #1494 (https://github.com/wso2/carbon-apimgt/commit/a031c40e48d28aafc715abc7fb45ae7ffbb48b9b)